nllb_serve/app.py

```diff
@@ -21,6 +22,9 @@
 device = torch.device(torch.cuda.is_available() and 'cuda' or 'cpu')
 log.info(f'torch device={device}')
 
+dicti = {"cat_Latn": "ca", "ces_Latn": "cs", "dan_Latn": "da", "nld_Latn": "nl","eng_Latn": "en","fin_Latn": "fi","fra_Latn": "fr","deu_Latn": "de",
```

Comment:
vague name. 
rename dict to `ssplit_langs`.

Also consider moving sentence splitting code to a separate file and import it here. The code will get complicated over time as we support more languages